### DISCUSSION TITLE
ERREUR 500 LISTENERS

### DISCUSSION THREAD

**Discussion by eamarelo** [PayGreen/documentation#37] [Q&A] [answered]
Bonjour, 

J'ai 2 listeners pour mon application, pour 2 environnements différents. => prod et preprod. 

Mes listeners sont similaires, celui en preprod fonctionne mais pas celui en prod. Avez vous une idée de pourquoi?

**Comment by AdrienBr** [comment, marked as answer]
Bonjour,
Pouvez-vous contacter le support en précisant vos identifiants de listener (pour la prod et la sandbox).

https://www.paygreen.io/contact

Merci,
Bonne journée
